src/examples/example.ts

```diff
@@ -4,7 +4,8 @@ import ComboControls from '../';
 const width = window.innerWidth;
 const height = window.innerHeight;
 const scene = new THREE.Scene();
-const camera = new THREE.PerspectiveCamera(60, width / height, 0.01, 100);
+// const camera = new THREE.PerspectiveCamera(60, width / height, 0.01, 100);
```

Comment:
Could you use this package https://www.npmjs.com/package/query-string in `example.ts` to be able to set a query parameter (`?orthographic`) to the URL to switch to orthograpic camera?